### ISSUE TITLE
Small background preview

### ISSUE THREAD

**Issue by EmmanuelHuazo** [coppolaemilio/dialogic#1539]
It can be hard to identify the backgrounds if we use sometype of nomenclature that cant be recognized in the first leters, so it would be awesome if Dialogic has a small preview of the backgrounds and the sprites/portraits on the timeline editor when we are selecting them.

![image](https://user-images.githubusercontent.com/79464276/235328271-c07bdefd-d7d6-493a-bc09-2984c640ff43.png)

![image](https://user-images.githubusercontent.com/79464276/235328273-027d9cef-e78a-4020-b09e-d77739e387df.png)

**Comment by EmmanuelHuazo** [comment]
I know I'm using an ultrawide monitor but I think there is a lot unused space in the background node that can be used for more info or options instead of expanding the node using double click.

![image](https://user-images.githubusercontent.com/79464276/235328642-9044f634-5b6c-4f58-8980-97009c81e497.png)

**Comment by coppolaemilio** [comment]
Yes, I had that issue of the small input fields as well. It will get addressed 👍
